### PR TITLE
DNS init container

### PR DESCRIPTION
**Pull request by joshwget**
Put the image contents in `images/` for now to track.

### CODE REVIEW

File: images/dns-init/dns-init.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+if [ "$APPEND" == "true" ]; then
+    echo $SEARCH >> /etc/resolv.conf
+else
+    echo -e "$SEARCH\n$(cat /etc/resolv.conf)" > /etc/resolv.conf
```
Comment (alena1108) [inline images/dns-init/dns-init.sh]: Won't we endup having 2 search lines here?

Comment (alena1108) [reply images/dns-init/dns-init.sh]: Also we should check if the domain already exists in /etc/resolv.conf before appending

**Comment by alena1108** [comment]
We might also need to set rancher dns in addition to dns search: https://github.com/rancher/plugin-manager/blob/master/events/start_handler.go#L82

**Comment by joshwget** [comment]
Going to redo this.

**Pull request closed without merging**